python/jupytergis_lab/jupytergis_lab/notebook/gis_document.py

```diff
@@ -109,23 +106,7 @@ def layer_tree(self) -> List[str | Dict]:
         """
         return self._layerTree.to_py()
 
-    def save_as(self, path: str | Path) -> None:
```

Comment:
This copies the document, whereas a "save as" operation would also change the current working document to alter the new file. We don't have the comms setup to do this yet. Restored the `export_to_qgis` public API function for now as well.